lms/static/sass/discussion/_discussion.scss

```diff
@@ -241,20 +228,17 @@ body.discussion {
 
   .discussion-column {
     @include float(right);
-    @include box-sizing(border-box);
+    box-sizing: border-box;
+    padding-left: ($baseline/2);
```

Comment:
Thanks @clrux. I've reported a new story to clean this up:

https://openedx.atlassian.net/browse/TNL-4709